### ISSUE TITLE
Add SteamGuard with OFF email-guard on acc

### ISSUE THREAD

**Issue by Bahrain-bag** [th3poli/steamguard#1]
I have error mobile.add_mobile_auth()
  File "C:\Users\elect\PycharmProjects\pythonProject3\venv\Lib\site-packages\steamguard\mobile.py", line 143, in add_mobile_auth
    status = res.get('status')
             ^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'

it arose when guard via mail was disabled on the account, I switched it manually and everything worked. Can you tell me how to fix this so as not to think about it later?

**Comment by th3poli** [comment]
Ahh sorry this is probably my fault. I will check it asap and repair. Thank you for your comment! I guess the request must be different for accounts without email steam guard.

**Comment by Bahrain-bag** [comment]
thank you, I have been looking for something like this for Python for a long time and your library was very useful to me

**Comment by th3poli** [comment]
Hey ;) The issue has been fixed in the latest release v1.0.6. Please update to the latest version and let me know if the problem persists. Now you can check if your account email Steam Guard is disabled:

`pip3 install steamguard --upgrade`

```python
mobile = SteamMobile('<steam login>', '<steam password>')

code_type = mobile.login()

if code_type == LoginConfirmType.none: # Check if Steam Guard is disabled
    mobile.confirm_login() # Login :)
```

**Comment by Bahrain-bag** [comment]
> Hey ;) The issue has been fixed in the latest release v1.0.6. Please update to the latest version and let me know if the problem persists. Now you can check if your account email Steam Guard is disabled:
> 
> `pip3 install steamguard --upgrade`
> 
> ```python
> mobile = SteamMobile('<steam login>', '<steam password>')
> 
> code_type = mobile.login()
> 
> if code_type == LoginConfirmType.none: # Check if Steam Guard is disabled
>     mobile.confirm_login() # Login :)
> ```

yeah it worked man
